Packs/GitHubITAdmin/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.py

```diff
@@ -0,0 +1,615 @@
+                            if j.get('value') != i.get('value'):
+                                j['value'] = i.get('value')
+                            if i.get('primary', None) is not None:
+                                j['primary'] = i.get('primary')
+                            else:
+                                if j.get('primary', None) is not None:
+                                    j['primary'] = j.get('primary')
+                            break
+
+                # add
+                new_email_list = []
+                for i in v:
+                    exist = False
+                    for j in existing_email_list:
+                        if i.get('type') == j.get('type', ''):
+                            exist = True
+                            break
+                    if not exist:
+                        new_email = {'type': i.get('type'),
+                                     'value': i.get('value')}
+                        if i.get('primary', None) is not None:
+                            new_email.update({'primary': i.get('primary')})
+                        new_email_list.append(new_email)
+                existing_email_list.extend(new_email_list)
+
+        elif type(v) == dict:
+            if k != SCIM_EXTENSION_SCHEMA:
+                map_changes_to_existing_user(existing_user.get(k), v)
+        else:
+            existing_user[k] = v
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client, args):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client: GitHub client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+
+    """result = client.http_request(method,url_suffix)
+    if res.status_code == 200:
+        return 'ok'
+    else:
+        raise Exception"""
+
+    uri = ""
+    res = client.http_request(method='GET', url_suffix=uri)
+    if res.status_code == 200:
+        return 'ok', None, None
+    else:
+        raise Exception(f"{res.status_code} - {res.text}")
+
+
+def get_user_command(client, args):
+    """
+        Returning user GET details and status of response.
+
+        Args:   demisto command line argument
+        client: GitHub
+
+        Returns:
+            success : success=True, id, email, login as username, details, active status
+            fail : success=False, id, login as username, errorCode, errorMessage, details
+    """
+
+    scim = verify_and_load_scim_data(args.get('scim'))
+    scim_flat_data = map_scim(scim)
+    user_id = scim_flat_data.get('id')
+    username = scim_flat_data.get('userName')
+    email = scim_flat_data.get('email')
+
+    input_type = None
+    if not (user_id or username or email):
+        raise Exception('You must provide either the id, email or login as userName of the user')
+
+    if user_id:
+        user_term = user_id
+        input_type = 'id'
+    else:
```

Comment:
```suggestion
    elif email:
```

might be more suited here

Reply:
Done